api/cluster/templater.go

```diff
@@ -215,6 +220,12 @@ func createTransformerSpec(modelService *models.Service, transformer *models.Tra
 	memoryLimit.Add(transformer.ResourceRequest.MemoryRequest)
 
 	envVars := transformer.EnvVars
+	if transformer.TransformerType == models.StandardTransformerType {
+		transformer.Image = t.standardTransformerConfig.ImageName
+		envVars = append(envVars, models.EnvVar{Name: transformerpkg.FeastServingAddressEnvName, Value: t.standardTransformerConfig.FeastServingAddress})
+		envVars = append(envVars, models.EnvVar{Name: transformerpkg.FeastServingPortEnvName, Value: string(t.standardTransformerConfig.FeastServingPort)})
+	}
```

Comment:
TODO: Add transformer config to env var as well

Reply:
It's not necessary since transformer config should already present in the envVars.